### PR TITLE
Release varnish cache 7.4.1

### PR DESCRIPTION
**Pull request by simonvik**

### CODE REVIEW

**Comment by simonvik** [comment]
If you have suggestions for a better description then please tell me!

How long will we support 7.4.0? Same as 7.4.1?

**Comment by Dridi** [comment]
It is technically 7.4.x that we support, so it supersedes the previous one.

File: R1/source/releases/index.rst [hunk 1]
```diff
@@ -12,6 +12,10 @@ Varnish Cache is released every 6 months.
     - Date
     - EOL Date
     - Download
+  * - :ref:`7.4.1 <rel7.4.1>`
+    - 2023-09-20
+    - 2024-09-20
+    - `varnish-7.4.1.tgz </downloads/varnish-7.4.1.tgz>`_
   * - :ref:`7.4.0 <rel7.4.0>`
     - 2023-09-15
     - 2024-09-15
```
Comment (Dridi) [inline R1/source/releases/index.rst:21]: 7.4.1 should replace 7.4.0 here.

Comment (simonvik) [reply R1/source/releases/index.rst]: Fixed.

File: R1/source/index.rst [hunk 2]
```diff
@@ -5,6 +5,13 @@ Varnish HTTP Cache
 
 What is happening
 -----------------
+2023-09-20 - Varnish 7.4.1 is released
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (Dridi) [inline R1/source/index.rst:9]: Missing blank line between headlines.

File: R1/source/releases/rel7.4.1.rst [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+Varnish Cache 7.4.1
+===================
+
+* Source download `varnish-7.4.1.tgz </downloads/varnish-7.4.1.tgz>`_
+
+* SHA256=874d837aaf49b8f2718cb60b8c8c7900e9ea10c264f218c88cd672d596f4b89f
+
+Varnish Cache 7.4.1 is a maintenance release that fixes a bug that
+made protected variables unreadable from backend subroutines.
```
Comment (Dridi) [inline R1/source/releases/rel7.4.1.rst:11]: ... that fixes a bug preventing protected headers to be read from several subroutines.

File: R1/source/releases/rel7.4.1.rst [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+More information:
+
+* List of most important `Changes in 7.4 <https://varnish-cache.org/docs/7.4/whats-new/changes-7.4.html>`_
+* Help on `Upgrading to Varnish 7.4 <https://varnish-cache.org/docs/7.4/whats-new/upgrading-7.4.html>`_
+* `Full changes.rst entry for 7.4.0 <https://github.com/varnishcache/varnish-cache/blob/7.4/doc/changes.rst#varnish-cache-741-2023-09-19>`_
```
Comment (Dridi) [inline R1/source/releases/rel7.4.1.rst:17]: "for 7.4.1"

**Review by Dridi — CHANGES_REQUESTED** [review]

File: R1/source/index.rst (hunk 2 above)
Comment (simonvik) [reply R1/source/index.rst]: Fixed.

File: R1/source/releases/rel7.4.1.rst (hunk 3 above)
Comment (simonvik) [reply R1/source/releases/rel7.4.1.rst]: Fixed.

File: R1/source/releases/rel7.4.1.rst (hunk 4 above)
Comment (simonvik) [reply R1/source/releases/rel7.4.1.rst]: Fixed.

File: R1/source/index.rst [hunk 5]
```diff
@@ -6,6 +6,14 @@ Varnish HTTP Cache
 What is happening
 -----------------
 
+2023-09-20 - Varnish 7.4.1 is released
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Varnish 7.4.1 has been released and can be found here: :ref:`rel7.4.1`
+
+This maintenance release fixes a bug that made protected
+variables unreadable from backend subroutines.
```
Comment (Dridi) [inline R1/source/index.rst:15]: My bad, I forgot to highlight this one, same wording as the other file.

File: R1/source/releases/index.rst [hunk 6]
```diff
@@ -12,10 +12,10 @@ Varnish Cache is released every 6 months.
     - Date
     - EOL Date
     - Download
-  * - :ref:`7.4.0 <rel7.4.0>`
+  * - :ref:`7.4.1 <rel7.4.1>`
     - 2023-09-15
     - 2024-09-15
```
Comment (Dridi) [inline R1/source/releases/index.rst:17]: New release date.

**Review by Dridi — CHANGES_REQUESTED** [review]

File: R1/source/releases/index.rst (hunk 6 above)
Comment (simonvik) [reply R1/source/releases/index.rst]: Fixed and i assume that the EOL date will be the same?

File: R1/source/index.rst (hunk 5 above)
Comment (simonvik) [reply R1/source/index.rst]: Fixed.

File: R1/source/releases/index.rst (hunk 6 above)
Comment (Dridi) [reply R1/source/releases/index.rst]: Correct.

**Review by Dridi — APPROVED** [review]

**Pull request merged**